families/sawtooth_validator_registry/protos/validator_registry.proto

```diff
@@ -1,4 +1,4 @@
-// Copyright 2017 Intel Corporation
+// Copyright 2019 Intel Corporation
```

Comment:
This should not be updated to a new year. I believe we either do not update the copyright or change the year to look like 2017, 2019. This would be a good question for @TomBarnes